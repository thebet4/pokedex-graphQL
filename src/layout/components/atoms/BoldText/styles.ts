import { colors } from '@config/theme'
import styled from 'styled-components/native'

export const Bold = styled.Text`
  color: ${colors.base[100]}
  font-family: "Inter_700Bold";
  text-transform: capitalize;
  font-size: 18px;
  line-height: 21px;
`
