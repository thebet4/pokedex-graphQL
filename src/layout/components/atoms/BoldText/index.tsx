import React, { ReactNode } from 'react'
import { TextStyle } from 'react-native'
import { Bold } from './styles'

interface ITitleProps {
  children: ReactNode
  props?: TextStyle
}

const BoldText = ({ children, props }: ITitleProps) => {
  return <Bold {...props}>{children}</Bold>
}

export default BoldText
