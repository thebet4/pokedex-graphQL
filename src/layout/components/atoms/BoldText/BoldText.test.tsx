import * as React from 'react'
import renderer from 'react-test-renderer'

import BoldText from '@layout/components/atoms/BoldText'

it(`renders <BoldText /> correctly`, () => {
  const tree = renderer.create(<BoldText>Pokemon</BoldText>).toJSON()

  expect(tree).toMatchSnapshot()
})
