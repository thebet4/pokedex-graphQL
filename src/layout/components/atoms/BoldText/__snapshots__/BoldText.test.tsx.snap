// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <BoldText /> correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Inter_700Bold",
        "fontSize": 18,
        "lineHeight": 21,
        "textTransform": "capitalize",
      },
    ]
  }
>
  Pokemon
</Text>
`;
